doc/data/messages/t/too-complex/bad.py

```diff
@@ -0,0 +1,54 @@
+fruit = {'apple': 1.1,
+         'pear': 0.8,
+         'banana': 1.2,
+         'mango': 3.5,
+         'peach': 0.5,
+         'melon': 4.9,
+         'orange': 2.0,
+         'strawberry': 2.5,
+         'mandarin': 2.3,
+         'plum': 0.5,
+         'watermelon': 6.4
+         }
```

Comment:
```suggestion
```
It's better if we create the dict in the good example imo

Reply:
I rewrote the code to include the dict content in the if/else construct in https://github.com/PyCQA/pylint/pull/8144#discussion_r1093158896